### ISSUE TITLE
[Bug Report] Lost data

### ISSUE THREAD

**Issue by DanSixtyNine** [stashapp/stash#3286]
I spent a few hours manually adding metadata for images and scenes. I also created new studios and performers for this. When I started stash the next time, the changes of the last hours were gone.

I have no idea why this happened so this might be hard to reproduce, however, any kind of help would be appreciated.

**Comment by DanSixtyNine** [comment]
@sleetx Yes, I made a backup shortly before, but this should not affect changes I made afterwards should it?

**Comment by Roycinger** [comment]
Are you running Stash in a Docker container with the DB saved inside the container?
